.github/workflows/sycl-linux-precommit.yml

```diff
@@ -79,6 +79,12 @@ jobs:
             target_devices: level_zero:gpu;opencl:gpu
             reset_intel_gpu: true
             extra_lit_opts: --param matrix-xmx8=True
+          - name: Intel Arc A-Series Graphics for L0 V2 adapter
+            runner: '["Linux", "arc"]'
+            image_options: -u 1001 --device=/dev/dri -v /dev/dri/by-path:/dev/dri/by-path --privileged --cap-add SYS_ADMIN
```

Comment:
agree with andrei we should combine this with the l0v1 soon

Reply:
Do you want me to do this in this PR? There is a chance that there are still some tests that might fail randomly due to too old driver version used on CI. Keeping the jobs separate for now could make it easier to deal with them.

Reply:
the tests current take 2m40s, so no can be in a separate pr